client/src/pages/chatlink/index.js

```diff
@@ -66,6 +67,17 @@ const App = () => {
             </div>
           )}
         </div>
+        <div
+          className={`${styles.sectionContribute} ${
+            darkMode === true ? styles.sectionDefault : styles.sectionDefaultLight
+          }`}
+        >
+          <div className={styles.title}>
+            Enter Chat by PIN
+            <TextField />
```

Comment:
As it's doesn't seem like re-usable generic text filed component, let's rename it to `PinInput` to avoid confusion in the future.